osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/monitoring/AbstractGetPowerQualityProfileHandler.java

```diff
@@ -183,6 +183,13 @@ private Map<CosemObject, List<CosemObject>> getProfilesToRead(
     this.addToMapIfNeeded(POWER_QUALITY_PROFILE_1, privateOrPublic, device, profilesWithObjects);
     this.addToMapIfNeeded(POWER_QUALITY_PROFILE_2, privateOrPublic, device, profilesWithObjects);
 
+    if (profilesWithObjects.isEmpty()) {
+      throw new IllegalArgumentException(
```

Comment:
Is een ProtocolAdapterException niet gepaster?

Reply:
Ik heb het gelijk gehouden aan de exceptie die de ObjectConfigService gooit als een object niet gevonden is. Dat is een IllegalArgumentException. In de PQ handler wordt gezocht naar 3 objecten, ipv 1. Als ze alle 3 niet gevonden worden leek mij dezelfde exceptie toepasselijk.